megalinter/descriptors/lua.megalinter-descriptor.yml

```diff
@@ -54,3 +57,37 @@ linters:
       vscode:
         - name: vscode-luacheck
           url: https://marketplace.visualstudio.com/items?itemName=dwenegar.vscode-luacheck
+
+  # SELENE
+  - linter_name: selene
+    linter_url: https://kampfkarren.github.io/selene/
+    linter_repo: https://github.com/Kampfkarren/selene
+    linter_rules_url: https://kampfkarren.github.io/selene/lints/index.html
+    linter_rules_configuration_url: https://kampfkarren.github.io/selene/usage/configuration.html
+    linter_rules_inline_disable_url: https://kampfkarren.github.io/selene/usage/filtering.html
+    linter_megalinter_ref_url: "no"
+    config_file_name: selene.toml
+    cli_config_arg_name: "--config"
+    examples:
+      - "selene myfile.lua"
+      - "selene --config chktexrc.toml myfile.lua"
+    install:
+      apk:
+        - g++
+      dockerfile:
+        - |
+          RUN cargo install --branch main --git https://github.com/Kampfkarren/selene selene
```

Comment:
@nvuillam where does cargo get set up in individual linters? As it isn't being built in the PR, we don't see it fail yet